libs/vscode-ui/feature-task-execution-form/src/lib/task-execution-form.component.scss

```diff
@@ -28,7 +28,6 @@ $offset-with-configurations: $offset-without-configurations +
   border: $input-border-width solid var(--vscode-settings-textInputBorder);
   color: #fff;
   text-align: center;
-  line-height: 30px;
```

Comment:
fixes this jankiness on the Run button focus border 
![Screen Shot 2021-04-29 at 9 42 35 AM](https://user-images.githubusercontent.com/2250413/116569745-4ab4ed80-a8cf-11eb-8b49-c5b62ea16039.png)
fixed:
![Screen Shot 2021-04-29 at 9 43 30 AM](https://user-images.githubusercontent.com/2250413/116569837-615b4480-a8cf-11eb-8e5f-77c6b37477f7.png)

Reply:
I hate when that happens 😅